server/server/data_loader/textfunctions.py

```diff
@@ -0,0 +1,38 @@
+import regex
+import unicodedata
+
+def pali_sort_key(input, _charvalue = {}):
```

Comment:
don't use `input` as a variable name. It is a bilt-in fucntion's name

Reply:
Oh dear, I'd actually mostly refactored the code in textfunctions (along with fixing some actual logic bugs) but accidentally pushed the old unfixed code.